codalab/model/bundle_model.py

```diff
@@ -96,10 +100,10 @@ def create_tables(self):
         self._create_default_groups()
         self._create_default_clients()
 
-    def do_multirow_insert(self, connection, table, values):
+    @staticmethod
```

Comment:
Turned all the little helper methods here that didn't use the self reference into static methods here to make it clearer.